tests/build.rs

```diff
@@ -0,0 +1,70 @@
+use assert_cmd::prelude::*;
+use assert_fs::prelude::*;
+use predicates::prelude::*;
+use rstest::rstest;
+use std::process::Command;
+
+mod constants;
+
+#[rstest]
+fn build_without_clone() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
Please add a doc comment for all new functions.

Reply:
Done